packages/bot-skeleton/src/services/tradeEngine/utils/helpers.js

```diff
@@ -165,9 +169,10 @@ const getBackoffDelayInMs = (error_obj, delay_index) => {
                 break;
             case 'OpenPositionLimitExceeded':
                 message_to_print = localize(
-                    'You already have an open position for this contract type, retrying in {{ delay }}s',
+                    'You already have an open position for this contract type Trade Type: {{ trade_type }}, retrying in {{ delay }}s',
```

Comment:
`for this contract type Trade Type:` should we keep both types (contract type and Trade Type) here? Or it should only be one of them?

Reply:
as per the card only Trade type is needed @shafin-deriv